internals/overlord/logstate/gatherer_test.go

```diff
@@ -63,7 +63,7 @@ func (s *gathererSuite) TestGatherer(c *C) {
 
 	testSvc.writeLog("log line #5")
 	select {
-	case <-time.After(5 * time.Millisecond):
+	case <-time.After(10 * time.Millisecond):
```

Comment:
These are merely maximum timeouts, right? IMO these should be significantly longer, like time.Second (we use 1s or 10s in other places). 10ms or even 100ms can easily be hit on a busy shared GitHub CI server. In the normal case the tests still run quickly, of course, so having a long-ish failure timeout is fine.

Reply:
changed them all to 1s now